### ISSUE TITLE
click my profile COLLECTIONS Cannot read property '_isDestroyed' of undefined     at destroy

### ISSUE THREAD

**Issue by seth-shi** [pixelfed/pixelfed#3218]
click my profile, Then click on COLLECTIONS, Page jammed

```
TypeError: Cannot read property '_isDestroyed' of undefined
    at destroy (vendor.js?id=a818775c694ed2a103e9:1)
    at b (vendor.js?id=a818775c694ed2a103e9:1)
    at b (vendor.js?id=a818775c694ed2a103e9:1)
    at b (vendor.js?id=a818775c694ed2a103e9:1)
    at b (vendor.js?id=a818775c694ed2a103e9:1)
    at b (vendor.js?id=a818775c694ed2a103e9:1)
    at b (vendor.js?id=a818775c694ed2a103e9:1)
    at w (vendor.js?id=a818775c694ed2a103e9:1)
    at vendor.js?id=a818775c694ed2a103e9:1
    at C (vendor.js?id=a818775c694ed2a103e9:1)
ze @ vendor.js?id=a818775c694ed2a103e9:1
We @ vendor.js?id=a818775c694ed2a103e9:1
Be @ vendor.js?id=a818775c694ed2a103e9:1
(anonymous) @ vendor.js?id=a818775c694ed2a103e9:1
Je @ vendor.js?id=a818775c694ed2a103e9:1
Promise.then (async)
Ve @ vendor.js?id=a818775c694ed2a103e9:1
tt @ vendor.js?id=a818775c694ed2a103e9:1
(anonymous) @ vendor.js?id=a818775c694ed2a103e9:1
INkZ.dn.update @ vendor.js?id=a818775c694ed2a103e9:1
INkZ.fe.notify @ vendor.js?id=a818775c694ed2a103e9:1
set @ vendor.js?id=a818775c694ed2a103e9:1
pn.set @ vendor.js?id=a818775c694ed2a103e9:1
switchMode @ profile.js?id=e0365f377fb6c574fda4:1
click @ profile.js?id=e0365f377fb6c574fda4:1
Ue @ vendor.js?id=a818775c694ed2a103e9:1
n @ vendor.js?id=a818775c694ed2a103e9:1
Xr.o._wrapper @ vendor.js?id=a818775c694ed2a103e9:1
```

**Comment by dansup** [comment]
@seth-shi Which instance?

**Comment by seth-shi** [comment]
https://user-images.githubusercontent.com/28035971/153327053-5b584617-0362-41c3-b68f-426aae38d42d.mp4

This error occurs when I force a refresh(ctrl+f5)

**Comment by dansup** [comment]
@seth-shi Fixed in #3227
